### PR TITLE
Added support for properties starts with 'scroll-snap'

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: src/plugins/scroll-snap.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default {
+  supportedProperty: (prop) => {
+    if (prop.startsWith('scroll-snap')) {
```
Comment (kof) [inline src/plugins/scroll-snap.js]: startsWith requires polyfill, lets just use substr.

Comment (AleshaOleg) [reply src/plugins/scroll-snap.js]: @kof ok

File: src/plugins/scroll-snap.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import prefix from '../prefix'
+
+export default {
+  supportedProperty: (prop) => {
+    if (prop.substring(0, 11) === 'scroll-snap') {
+      if (prefix.js === 'ms') {
```
Comment (kof) [inline src/plugins/scroll-snap.js]: you want to add prefix for IE only? Why not to exit early before any code needs to bee executed then? On line 2

Comment (AleshaOleg) [reply src/plugins/scroll-snap.js]: @kof, yes, prefix needed only, for IE/Edge. Didn't understood how i can this one code better:(

Comment (kof) [reply src/plugins/scroll-snap.js]: This plugin seems to do nothing when browser is not IE, why not to do an early exit on line 2 then? prefix.js is already available there.

Comment (AleshaOleg) [reply src/plugins/scroll-snap.js]: Ah, no. It also should return value if browser is Safari. And should be prefixed, if IE.

Comment (AleshaOleg) [reply src/plugins/scroll-snap.js]: @kof, so that's why I wrote it. If no Safari support needed, there is no needed on this plugin

Comment (AleshaOleg) [reply src/plugins/scroll-snap.js]: may be, we need to add safari 11 to our test suite in `browsers.json`, to check this changes?

File: src/plugins/scroll-snap.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        return `${prefix.css}${prop}`
+      }
+      return prop
```
Comment (kof) [inline src/plugins/scroll-snap.js]: You mean this line? Isn't it by default?

Comment (AleshaOleg) [reply src/plugins/scroll-snap.js]: For Safari 11, it shows errors for properties starts with 'scroll-snap'. And if no prefix for IE - it also shows error. So, I should check if it's IE.
Yes, it not return this props by default in browser. That's why I should do this plugin.

**Comment by kof** [comment]
merged!

**Pull request merged**